### PR TITLE
fix: capabilities response handling in LambdaTest

### PR DESCRIPTION
**Pull request by swngarg-lt**

### CODE REVIEW

**Review by KazuCocoa — COMMENTED** [review]
Could you explain more details?
`/sessions` endpoint used in https://github.com/appium/appium-inspector/blob/c92cedf7d94354c446e98d12519f22b0e9e962ec/app/renderer/actions/Session.js#L828-L839 is not defined in W3C, so could you explain the expected response etc what is W3C as you described in the PR title?

Current each session used in AttachToSession was each object in `value` like the below format.
```
{
  "status": 0, "session_id": null, 
  "value": [{"id": "session id 1", "capabilities": {...}}, {"id": "session id 2", "capabilities": {...}}]
}
```

File: app/renderer/components/Session/AttachToSession.js [hunk 1]
```diff
@@ -25,26 +25,33 @@ function formatCapsBrowserstack (caps) {
...
   }
-  let importantCaps = [caps.deviceName, caps.platformName, caps.platformVersion];
+  let importantCaps = [deviceName, caps.platformName, caps.platformVersion];
   return importantCaps.join(', ').trim();
 }
 
 export default class AttachToSession extends Component {
 
+  getSessionId (session) {
+    return session.sessionId === undefined || session.sessionId === 'undefined' ? session.id : session.sessionId;
```
Comment (KazuCocoa) [inline app/renderer/components/Session/AttachToSession.js:43]: What situations an be `'undefined'` string? So far, Appium's `/sessions` endpoint returns the list of session id (as `id`) and `capabilities`. So we need to understand this `sessionId`.

**Comment by KazuCocoa** [comment]
Is this PR no longer active?

File: app/renderer/components/Session/AttachToSession.js [hunk 2]
```diff
@@ -25,26 +25,33 @@ function formatCapsBrowserstack (caps) {
 }
 
 function formatCapsLambdaTest (session) {
-  let caps;
-  if (session.capabilities) {
+  let caps = session, deviceName = session.deviceName;
```
[the remark this answers is not in the document]
Comment (rohithmone-lt) [reply app/renderer/components/Session/AttachToSession.js:28]: Sure

[the remark this answers is not in the document]
Comment (rohithmone-lt) [reply app/renderer/components/Session/AttachToSession.js]: Done

**Comment by KazuCocoa** [comment]
Could you add description? So, the title was incorrect, to support your own `/sessions` endpoint definition?

File: app/renderer/components/Session/AttachToSession.js [hunk 3]
```diff
@@ -24,16 +24,9 @@ function formatCapsBrowserstack (caps) {
...
-  if (session.capabilities) {
-    caps = session.capabilities;
-  } else if (session.desired) {
-    caps = session.desired;
-  } else {
-    caps = session;
-  }
-  let importantCaps = [caps.deviceName, caps.platformName, caps.platformVersion];
+function formatCapsLambdaTest (caps) {
+  let deviceName = caps.desired ? caps.desired.deviceName : caps.deviceName;
```
Comment (KazuCocoa) [inline app/renderer/components/Session/AttachToSession.js:28]:
```suggestion
  const deviceName = caps.desired ? caps.desired.deviceName : caps.deviceName;
```

File: app/renderer/components/Session/AttachToSession.js [hunk 4]
```diff
@@ -24,16 +24,9 @@ function formatCapsBrowserstack (caps) {
...
+  let importantCaps = [deviceName, caps.platformName, caps.platformVersion];
```
Comment (KazuCocoa) [inline app/renderer/components/Session/AttachToSession.js:29]:
```suggestion
  const importantCaps = [deviceName, caps.platformName, caps.platformVersion];
```

**Review by KazuCocoa — COMMENTED** [review]
So... is this PR changed to modify a bit?

Still no description and maybe not matched with the title, so I'm a bit confused.

File: app/renderer/components/Session/AttachToSession.js (hunk 4 above)
Comment (swngarg-lt) [reply app/renderer/components/Session/AttachToSession.js:32]: When we were sending w3c as true, response was changing in session capabilities. Applied a fix, please have a look.

Comment (swngarg-lt) [reply app/renderer/components/Session/AttachToSession.js:32]: @KazuCocoa

Comment (KazuCocoa) [reply app/renderer/components/Session/AttachToSession.js:32]: Please fix the lint errors and let/const things i already commented

Comment (rohithmone-lt) [reply app/renderer/components/Session/AttachToSession.js]: Done @KazuCocoa

File: app/renderer/components/Session/AttachToSession.js (hunk 1 above)
Comment (rohithmone-lt) [reply app/renderer/components/Session/AttachToSession.js]: Fixed this

File: app/renderer/components/Session/AttachToSession.js (hunk 3 above)
Comment (rohithmone-lt) [reply app/renderer/components/Session/AttachToSession.js]: Done

File: app/renderer/components/Session/AttachToSession.js (hunk 4 above)
Comment (rohithmone-lt) [reply app/renderer/components/Session/AttachToSession.js]: Please check now @KazuCocoa

Comment (KazuCocoa) [reply app/renderer/components/Session/AttachToSession.js]: Hopefully I'll take a look tonight

**Review by KazuCocoa — APPROVED** [review]

**Pull request merged**